worlds/satisfactory/Options.py

```diff
@@ -0,0 +1,574 @@
+    "Bundle: Iron Rod",
+    "Bundle: Wire",
+    "Single: Reinforced Iron Plate",
+    "Single: Reinforced Iron Plate",
+    "Single: Reinforced Iron Plate",
+    "Single: Reinforced Iron Plate",
+    "Single: Reinforced Iron Plate",
+    "Single: Reinforced Iron Plate",
+    "Single: Reinforced Iron Plate",
+    "Single: Reinforced Iron Plate",
+    "Single: Reinforced Iron Plate",
+    "Single: Reinforced Iron Plate",
+    "Bundle: Cable",
+    "Bundle: Iron Ore"
+]
+
+_default_starting_items = _skip_tutorial_starting_items + [
+    "Bundle: Iron Ingot",
+    "Bundle: Copper Ingot",
+    "Bundle: Concrete",
+    "Bundle: Solid Biofuel", # user's choice if they want to hold onto it for chainsaw or burn it right away
+    "Building: Blueprint Designer",
+    "Expanded Toolbelt",
+    "Inflated Pocket Dimension",
+    "Building: Personal Storage Box"
+]
+
+_default_plus_foundations_starting_items = _default_starting_items + [
+    "Building: Foundation",
+    "Building: Half Foundation"
+]
+
+_explorer_starting_items = _default_plus_foundations_starting_items + [
+    "Single: Parachute",
+    "Single: Blade Runners",
+    "Single: Object Scanner",
+    "Single: Boom Box",
+    "Expanded Toolbelt",
+    "Inflated Pocket Dimension"
+]
+
+_foundation_lover_starting_items = _default_plus_foundations_starting_items + [
+    "Bundle: Iron Plate", "Bundle: Iron Plate", "Bundle: Iron Plate",
+    "Bundle: Concrete", "Bundle: Concrete", "Bundle: Concrete"
+]
+
+class StartingInventoryPreset(ChoiceMap):
+    """
+    What resources (and buildings) the player should start with in their inventory.
+    If you want more control, visit the Weighted Options page or edit the YAML directly.
+
+    - **Barebones**: Nothing but the default xeno zapper and buildings.
+    - **Skip Tutorial Inspired**: Inspired by the items you would have if you skipped the base game's tutorial.
+    - **Archipelago**: The starting items we think will lead to a fun experience.
+    - **Foundations**: 'Archipelago' option, but also guaranteeing that you have foundations unlocked at the start.
+    - **Foundation Lover**: You really like foundations.
+    - **Explorer**: 'Foundations' option plus one set of early exploration equipment (Parachute, Blade Runners, Object Scanner, Boom Box).
+    """
+    display_name = "Starting Goodies Presets"
+    choices = {
+        "Barebones": [], # Nothing but the xeno zapper
+        "Skip Tutorial Inspired": _skip_tutorial_starting_items,
+        "Archipelago": _default_starting_items,
+        "Foundations": _default_plus_foundations_starting_items,
+        "Foundation Lover": _foundation_lover_starting_items,
+        "Explorer": _explorer_starting_items
+    }
+    default = "Archipelago"
+
+class ExplorationCollectableCount(Range):
+    """
+    Does nothing if *Exploration Collectables* goal is not enabled.
+
+    Collect this amount of Mercer Spheres, Somersloops, Hard Drives, Paleberries, Beryl Nuts and Bacon Agarics each to finish.
```

Comment:
```suggestion
    Collect this amount of Mercer Spheres, Somersloops, Hard Drives, Paleberries, Beryl Nuts, and Bacon Agarics each to finish.
```
Oxford comma my beloved